saleor/product/migrations/0130_migrate_cost_price_to_channel_listing.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.1 on 2020-09-29 07:32
```

Comment:
Maby we can update migration witch introduces `ProductVariantChannelListing` instead of creating new migration? At the end of work on multichannel, we should merge as many migrations as possible.

Reply:
I think, we should merge migration in seperate task.